modules/structs/secret.go

```diff
@@ -5,11 +5,24 @@ package structs
 
 import "time"
 
-// User represents a secret
+// Secret represents a secret
 // swagger:model
 type Secret struct {
+	ID int64 `json:"id"`
```

Comment:
Could we not expose the id to users?

Reply:
for PATH and DELETE API.

Reply:
3223b80 and [f67ea2e](https://github.com/go-gitea/gitea/pull/26566/commits/f67ea2e58b86a759c02aee716ff843fbc927236b)